src/main/java/use_cases/add_edit_expenses_use_case/ExpenseUCI.java

```diff
@@ -157,7 +149,7 @@ public ExpenseOD editExpenseInMonth(ExpenseID expenseIDEdit) throws EntityExcept
      * @throws NoSuchElementException thrown when couldn't find Expense with String name.
      */
         public Expense findExpense(ArrayList<Expense> expenseData, String name)throws NoSuchElementException{
-            if(expenseData != null) {
+            if(!expenseData.isEmpty()) {
```

Comment:
Why did you make this switch? If the program runs null.isEmpty() it will throw a Null Pointer Exception.

Reply:
Yup that is a mistake tks!

Reply:
No worries!